### PR TITLE
Fixing aux parameter in ramping constraint

### PR DESCRIPTION
**Pull request by GabrielSoto-INL**
--------
Pull Request Description
--------
##### What issue does this change request address?
#357 

##### What are the significant changes in functionality due to this change request?
Auxiliary parameter for binary variables (used only when ramp frequencies are selected by user) is now a smaller value scaling with limit. This prevents the lower bound for ramping to be unintentionally constrained. 

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/HERON/wiki/Code-Standards) for details.
- [x] 4. Automated Tests should pass.
- [x] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large tes.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added, the the analytic documentation must be updated/added.
- [x] 9. If any test used as a basis for documentation examples have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: src/dispatch/PyomoRuleLibrary.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def ramp_rule_down(prod_name, r, limit, neg_cap, t, m, bins=None) -> bool:
       # dq is negative, - limit is negative
       return delta >= - limit
   else:
-    eps = 1.0 # aux parameter to force binaries to behave, TODO needed?
+    eps = 1e-6*limit # aux parameter to force binaries to behave
```
Comment (dylanjm) [inline src/dispatch/PyomoRuleLibrary.py]: Can expound on your comment a bit more. Things like why would it not behave and why choose such a small number.

**Review by dylanjm — CHANGES_REQUESTED** [review]
Minor request for more clarification in comment

File: src/dispatch/PyomoRuleLibrary.py (hunk 1 above)
Comment (GabrielSoto-INL) [reply src/dispatch/PyomoRuleLibrary.py]: I added more explanation to the comments here. Additionally, added a xlsx file to the relevant regression test with some sanity checking for the signs, here is a screenshot: 
image

**Review by dylanjm — APPROVED** [review]
Changes are good to merge.

**Pull request merged**